### PR TITLE
Github: Add ARM emitter tests to CI

### PR DESCRIPTION
**Pull request by Sonicadvance1**

### CODE REVIEW

**Review by lioncash — APPROVED** [review]

File: CMakeLists.txt [hunk 1]
```diff
@@ -30,6 +30,7 @@ option(ENABLE_CCACHE "Enables ccache for compile caching" TRUE)
 option(ENABLE_TERMUX_BUILD "Forces building for Termux on a non-Termux build machine" FALSE)
 option(ENABLE_VIXL_SIMULATOR "Forces the FEX JIT to use the VIXL simulator" FALSE)
 option(ENABLE_VIXL_DISASSEMBLER "Enables debug disassembler output with VIXL" FALSE)
+option(COMPILE_VIXL_DISASSEMBLER "Compiles the vixl disassembler in to vixl" FALSE)
```
Comment (neobrain) [inline CMakeLists.txt:33]: Is there a typo here? Not sure what compiling the vixl X in to vixl means, and it doesn't seem to be related to any of the other changes here.
